pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -343,6 +467,35 @@ func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: NodeRoleBindingName,
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: NodeRoleName,
+			},
+			Groups: util.NewStringSet(NodesGroup),
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: NodeProxyRoleBindingName,
+			},
+			RoleRef: kapi.ObjectReference{
+				Name: NodeProxyRoleName,
+			},
+			// Allow node identities to run node proxies
+			Groups: util.NewStringSet(NodesGroup),
```

Comment:
reduction of downstream churn (install scripts are specifying groups directly now), flexibility to remap stuff without regenerating certs later... the separation into roles was the main thing I wanted